src/nibpkg/kmers.nim

```diff
@@ -159,24 +154,24 @@ proc dna_to_kmers*(sq: Dna; k: int): pot_t =
         else:
             ##  advance the window beyond the unknown character
             lk = 0
-            inc(i, k)
-            inc(forward_kmer.pos, k)
+            inc(forward_kmer.pos)
```

Comment:
This is a functional change. It was incrementing `i` by `k`, but now it's `pos` by `1`. I think this was the purpose of using `while i` instead of `for c in` -- to skip ahead faster.

Do you think incrementing by `k` was a bug? If not, I think the old way was probably faster overall.